### PR TITLE
Add dlrm bench

### PR DESCRIPTION
**Pull request by satyaog**

### CODE REVIEW

**Comment by satyaog** [comment]
Last commit is not required

File: benchmarks/dlrm/prepare.py
Comment (breuleux) [inline benchmarks/dlrm/prepare.py:1]: Remove file.

File: benchmarks/dlrm/requirements_repo.in [hunk 1]
```diff
@@ -0,0 +1,10 @@
+future
+numpy
+onnx
+pydot
+torch
+torchviz
+scikit-learn
+tqdm
+torchrec-nightly
+torchx-nightly
```
Comment (breuleux) [inline benchmarks/dlrm/requirements_repo.in:10]: Does it really need nightly?

File: milabench/config.py [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
 from .fs import XPath
 from .merge import self_merge
 
 
 def parse_config(config_file, base=None):
+    benchmark = None
+    # It might be a yaml
+    if not XPath(config_file).exists():
+        benchmark = yaml.safe_load(io.StringIO(config_file))
+        config_file = benchmark["config_file"]
```
Comment (breuleux) [inline milabench/config.py:13]: Not sure what the purpose of this is.

Comment (satyaog) [reply milabench/config.py:13]: only useful when in `milabench dev` as you don't need to specify again the select but not necessary

File: benchmarks/dlrm/requirements_repo.in (hunk 1 above)
Comment (satyaog) [reply benchmarks/dlrm/requirements_repo.in:10]: From their requirements yes but it's possible that it was prior the release of pt 2.0

**Review by breuleux — APPROVED** [review]

**Pull request merged**